tests/test_fault_interpretation.py

```diff
@@ -0,0 +1,142 @@
+import pytest
+
+import uuid
+import resqpy.organize as rqo
+from resqpy.model import Model
+
+
+def test_is_equivalent_no_other(tmp_model):
+    #Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+        domain = "depth",
+        is_normal = True,
+        maximum_throw = 3,
+    )
+    #Act
+    result = fault_interp.is_equivalent(other = None)
+
+    #Assert
+    assert result is False
+
+
+def test_is_equivalent_other_not_FaultInterpretation(tmp_model):
+    # Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+        domain = "depth",
+        is_normal = True,
+        maximum_throw = 3,
+    )
+    # Act
+    result = fault_interp.is_equivalent(other = 55)
+
+    # Assert
+    assert result is False
+
+
+def test_is_equivalent_is_other(tmp_model):
+    # Arrange
+    fault_interp = rqo.FaultInterpretation(
+        tmp_model,
+        title = 'test_fault_interpretation',
+    )
+    # Act
+    other = fault_interp
+    result = fault_interp.is_equivalent(other = other)
+
+    # Assert
+    assert result is True
+
+
+def test_is_equivalent_is_sameUUIDs(tmp_model):
+
+    # Arrange
+    object_uuid = uuid.uuid4()
```

Comment:
It might be better to use `object_uuid = resqpy.olio.new_uuid()` instead.

Reply:
Done